### ISSUE TITLE
Add Car Icons

### ISSUE THREAD

**Issue by volik25** [primefaces/primeicons#388]
Can you add please Car Icons

As [Local Shipping](https://fonts.google.com/icons?selected=Material+Icons&icon.query=local+shipping), [Directions Car](https://fonts.google.com/icons?selected=Material+Icons&icon.query=Directions+Car) and same other?

Thank you!

**Comment by tbeseda** [comment]
+1 the truck would be helpful

**Comment by eironman** [comment]
Agree! It would be great to have like a transportation set of icons: bus, truck, car, plane, boat

**Comment by mhirst1992** [comment]
Agree. Full transportation suite would be useful as mentioned by @eironman

**Comment by StefanoZambolin** [comment]
+1
